### ISSUE TITLE
Bug: XAML Controls Gallery: Accessibility: [MAS 2.4.7]: The keyboard focus is not visible on the Pager menu (Carousals) of the "PipsPager Window"

### ISSUE THREAD

**Issue by v-amnat** [microsoft/microsoft-ui-xaml#4297]
**Tags**
Accessibility, A11yCT, A11yMAS, A11yMediumImpact, MAS2.4.7, Keyboard, Sev3, XAML Controls Gallery (Debug), Win32, Wipro

**Steps to reproduce the bug**
1. Launch the XAML Controls Gallery (Debug) App
2. Go to the Scrolling-> Pips Pager.
3. Hit on Tab key to traverse the "Pager menu (Carousals)"
4. Observe the keyboard focus on the "Pager menu (Carousals)"

**Actual behavior**
The keyboard focus focus traverses the carousals' pager menu, but its not visible on the Pager menu (Carousals) of the "PipsPager Window"
Note: Although Narrator focus is visible on the same.

**Expected behavior**
The keyboard focus is not visible on the Pager menu (Carousals) of the "PipsPager Window"

**User Impact**
If keyboard focus is not visible on the on carousals' pager menu, then it could be difficult for the Keyboard user to understand its availability. 

**Attachment**
[KB-Scrolling-PipsPager.zip](https://github.com/microsoft/microsoft-ui-xaml/files/6019810/KB-Scrolling-PipsPager.zip)

**Version Info**
Microsoft OS: 21313.1000
App: XAML Controls Gallery (Debug) (Version 2.1.0.0)

**Comment by YuliKl** [comment]
Moving back to WinUI repo because this smells more like a controls problem rather than an app problem to me. I'm also seeing strange behavior with arrow keys on the first PipsPager sample (right arrow eventually moves focus off PipsPager to the "Related Controls" section). @StephenLPeters @gabbybilka FYI.

**Comment by gabbybilka** [comment]
@YuliKl the bug for lack of keyboard focus is caused by the extra tab stop to be fixed by this PR: #4107
 
@beervoley I was also able to replicate the behavior mentioned by Yulia above in XCG. I'm wondering if this is related to #4246.

**Comment by gabbybilka** [comment]
@beervoley Did #4107 fix the keyboard focus bug? Is there any update to investigation for the scroll behavior bug?

**Comment by beervoley** [comment]
@gabbybilka yes, it did fix the bug :) 
there's no update in work on scroll behaviour bug - but I've found the reason and will fix it in the following weeks:)

**Comment by gabbybilka** [comment]
@gabbybilka Thanks! Can you open a bug for the scroll behavior bug/link it here? Then I will close this issue as fixed.

**Comment by gabbybilka** [comment]
Yes, confirmed that #4246 covers the bug that @YuliKl mentioned. Will now close this issue as it was fixed by #4107.
